ui-v2/app/styles/components/service-list/skin.scss

```diff
@@ -0,0 +1,39 @@
+%service-list > a > span:nth-child(2) {
+  font-size: 1.125rem;
+  font-weight: $typo-weight-medium;
+}
+%service-list .empty {
+  @extend %with-minus-square-fill-color-mask, %as-pseudo;
+  background-color: #7c8797;
+}
+%service-list .kubernetes {
+  @extend %with-kubernetes-logo-color-icon, %as-pseudo;
+}
+%service-list .terraform {
+  @extend %with-terraform-logo-color-icon, %as-pseudo;
+}
+%service-list .nomad {
+  @extend %with-nomad-logo-color-icon, %as-pseudo;
+}
+%service-list .consul {
+  @extend %with-consul-logo-color-icon, %as-pseudo;
+}
+%service-list .aws {
+  @extend %with-logo-aws-color-icon, %as-pseudo;
+}
+%service-list .passing {
+  @extend %with-check-circle-fill-color-mask, %as-pseudo;
+  background-color: $green-500;
+}
+%service-list .warning {
+  @extend %with-alert-triangle-color-mask, %as-pseudo;
+  background-color: $orange-500;
+}
+%service-list .critical {
+  @extend %with-cancel-square-fill-color-mask, %as-pseudo;
+  background-color: $red-500;
+}
+%service-list .proxy::before {
+  @extend %with-swap-horizontal-mask, %as-pseudo;
+  background-color: $gray-500;
+}
```

Comment:
Wondered if there is anything to pick here, that means you only add the `%as-pseudo` once? What is it the `a > span:first-child`?  No biggie, but if you'd prefer that?